### ISSUE TITLE
Snowflake format: DDL COMMENT column goes into new line

### ISSUE THREAD

**Issue by PKRacha** [sql-formatter-org/sql-formatter#636]
**Describe the Feature**
In snowflake, we can provide COMMENT to the columns in DDL. Currently, when we format, the COMMENT with description comes to the next line. But, this should be on the same line.
Same case for a view defintion.
e.g.:
```CREATE TABLE IF NOT EXISTS table1 (id number(38, 0) COMMENT 'Identifier column', stringcol string COMMENT 'string column') CLUSTER BY (id) COMMENT = 'info about table1';```

current result
```
CREATE TABLE IF NOT EXISTS 
             table1 (
                          id number(38, 0) 
                          COMMENT 'Identifier column', 
                          stringcol string 
                          COMMENT 'string column'
             ) 
CLUSTER BY 
             (id) 
COMMENT = 'info about table1'
;
```

Expected result
```
CREATE TABLE IF NOT EXISTS  table1 (
    id number(38, 0) COMMENT 'Identifier column', 
    stringcol string COMMENT 'string column'
) CLUSTER BY (id) COMMENT = 'info about table1'
;
```

**Why do you want this feature?**
Snowflake DB has new options in their syntax, which are not available in previous database. To support those, we need to update to support those.

**Comment by nene** [comment]
Thanks for bringing up this issue.

There's currently quite a bit of inconsistency how the `COMMENT` keyword is handled in various dialects. In some (e.g. Snowflake and Postgres) a newline is added before the `COMMENT` keyword, like:

```sql
CREATE TABLE
  table1 (
    id number (38, 0) COMMENT 'Identifier column',
    stringcol string COMMENT 'string column'
  ) COMMENT 'info about table1';
```

While in others (e.g. MySQL, Hive), no newline gets added:

```sql
CREATE TABLE
  table1 (
    id number(38, 0) COMMENT 'Identifier column',
    stringcol string COMMENT 'string column'
  ) COMMENT 'info about table1';
```

The real motivation behind this was the formatting of `ALTER TABLE` statements, like:

```sql
ALTER TABLE foo
ALTER COLUMN col1
COMMENT 'my column comment';
```

But because of the deficiencies in how sql-formatter works, it can't distinguish between `COMMENT` appearing inside `CREATE TABLE` or `ALTER TABLE`, so it's either always adding newline or never.

I think it would be better to drop this behavior as that `ALTER TABLE` + `COMMENT` clause is IMHO kinda rare compared to `CREATE TABLE` + `COMMENT`. And not adding the newline won't really result in clearly worse formatting:

```sql
ALTER TABLE foo
ALTER COLUMN col1 COMMENT 'my column comment';
```

**Comment by nene** [comment]
Fixed in `master`.

**Comment by nene** [comment]
Published the fix in 13.0.0

**Comment by PKRacha** [comment]
Thanks, @nene. When will these changes are updated in VS Code extenstion?

**Comment by nene** [comment]
Thanks for reminding. Released in 2.1.1 version of the extension.

**Comment by PKRacha** [comment]
Thank you, @nene. I see the version update in VS Code and also the updates for the COMMENT.
